lobby-db/drop_db

```diff
@@ -6,6 +6,8 @@
 
 DB=lobby_db
 PSQL="psql -h localhost -U lobby_user -w -d postgres"
+
+echo "select pg_terminate_backend(pid) from pg_stat_activity where datname='$DB';" | $PSQL
```

Comment:
Interesting, sql really makes things weird if no data needs to actually be fetched ^^